### PR TITLE
Removed account info auto-fill from account selector (#2596)

### PR DESCRIPTION
**Pull request by chrlschwb**
#2596
Removed <AccountInfo> component from 
pioneer\packages\ui\src\accounts\components\SelectAccount\OptionAccount.tsx

### CODE REVIEW

**Comment by traumschule** [comment]
#2596

**Comment by traumschule** [comment]
This removed the account address. Positioning issue is unrelated.

missing-address

File: packages/ui/src/accounts/components/SelectAccount/OptionAccount.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ export const OptionAccount = ({ option, isForStaking }: Props) => {
 
   return (
     <>
-      <AccountInfo account={option} locked={isLocked} />
+      {/*<AccountInfo account={option} locked={isLocked} />  // Address info commented out */}
```
Comment (traumschule) [inline packages/ui/src/accounts/components/SelectAccount/OptionAccount.tsx:26]: This breaks account selection and does not fix the issue.

**Review by traumschule — CHANGES_REQUESTED** [review]

**Review by oleksanderkorn — CHANGES_REQUESTED** [review]
Worse to try `autocomplete="none"`:
image

Previous solution `autocomplete="new-password"` have been working but not anymore in the latest chrome.

There have been an ongoing discussion and this may still break in future versions:
https://stackoverflow.com/questions/15738259/disabling-chrome-autofill

**Comment by traumschule** [comment]
I don't think this approach is the right solution. Feel free to reopen with a screenshot.

**Pull request closed without merging**